rpc/tendermint/test/common.go

```diff
@@ -32,7 +32,7 @@ func TestWrapper(runner func() int) int {
 func DebugMain() {
 	t := &testing.T{}
 	TestWrapper(func() int {
-		testBroadcastTx(t, "HTTP")
```

Comment:
No, I think putting a short comment above the function is more than adequate to argue for it's existence (referencing delve as a debugging tool)